packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoTableHeaderRow.tsx

```diff
@@ -83,36 +95,73 @@ export const WfoTableHeaderRow = <T extends object>({
                                 </WfoTableHeaderCell>
-                            </th>
-                        );
-                    }
+                                {onUpdateColumWidth && (
+                                    <div
+                                        css={dragAndDropStyle}
+                                        draggable={true}
+                                        onDragStart={(e) => {
+                                            startDragPosition = e.clientX;
+                                            if (headerRowRef.current) {
+                                                const thElement =
+                                                    headerRowRef.current.querySelector(
+                                                        `th[data-field-name="${fieldName}"]`,
+                                                    ) as HTMLTableCellElement;
+                                                startWidth =
+                                                    thElement.getBoundingClientRect()
+                                                        .width;
+                                            }
+                                        }}
+                                        onDragEnd={(e) => {
+                                            if (
+                                                headerRowRef.current &&
+                                                onUpdateColumWidth
+                                            ) {
+                                                const travel =
+                                                    e.clientX -
+                                                    startDragPosition;
+                                                const newWidth =
+                                                    startWidth + travel;
 
-                    // Control column
-                    return (
-                        <th
-                            key={key}
-                            colSpan={columnConfig.numberOfColumnsToSpan}
-                            css={[
-                                ...toOptionalArrayEntry(
-                                    cellStyle,
-                                    !columnConfig.disableDefaultCellStyle,
-                                ),
-                                setWidth(columnConfig.width),
-                            ]}
-                        >
-                            <div>{columnConfig.label?.toString() || ''}</div>
+                                                onUpdateColumWidth(
+                                                    fieldName,
+                                                    newWidth,
+                                                );
+                                            }
+                                        }}
+                                    >
+                                        &nbsp;
+                                    </div>
```

Comment:
Was it not possible to use the boundary between 2 cells as point to drag on? I think that is why it becomes impossible to resize very narrow colums: colums where the header text cannot fit in

Reply:
I have since fixed the resizing of smaller columns and put in a minimum width. There is a draghandler component now that makes it possible/easier to show a movable line